html/index.html

```diff
@@ -1,11 +1,71 @@
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
-    
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>MALNUTRITION</title>
+        <link rel="stylesheet" href="../css/style.css">
+    </head>
+    <body>
+        
+        <div class="container">
+            <img src="../images/malnutrition pix.jpg" alt="">
+            <ul>
+                <li>
+                    <a href="#cons"> <strong>Consequences of Malnutrition</strong> </a>
+                </li>
+                <li>
+                    <a href="#solution"> <strong>Solution to Malnutrition in Africa</strong> </a>
+                </li>
+            </ul>
+            <h1>Malnutrition in Africa: Causes, Effects, and Solutions</h1>
+            <a href="#"></a>
+                <p> 
+                    Malnutrition is a significant public health concern in Africa, where millions of people are affected by its consequences. <br> The continent is home to some of the highest rates of malnutrition in the world, and it affects children, pregnant women, <br> and adults alike. Malnutrition is a complex problem that is caused by various factors, including poverty,  poor healthcare, <br> inadequate education, and environmental factors. In this article, we will explore the causes, effects, and solutions to <br> malnutrition in Africa.
```

Comment:
Kindly avoid frequent use of break tags within paragraph elements. Rather, style with margins.